src/common/unknown.js

```diff
@@ -10,27 +10,26 @@ import type {Action} from "./unknown";
 import type {Never} from "reflex";
 */
 
-export const warn =
-  (...params/*:Array<any>*/)/*:Task<Never, Action>*/ =>
-  new Task((succeed, fail) => {
+export function warn(...params: Array<any>): Task<Never, Action> {
+  return new Task((succeed, fail) => {
     console.warn(...params);
   });
+}
 
-export const log =
-  (...params/*:Array<any>*/)/*:Task<Never, Action>*/ =>
-  new Task((succeed, fail) => {
+export function log(...params: Array<any>): Task<Never, Action> {
+  return new Task((succeed, fail) => {
     console.log(...params);
   });
+}
 
-export const error =
-  (...params/*:Array<any>*/)/*:Task<Never, Action>*/ =>
-  new Task((succeed, fail) => {
+export function error(...params: Array<any>): Task<Never, Action> {
+  return new Task((succeed, fail) => {
     console.error(...params);
   });
+}
 
 
-export const update = /*::<model, action>*/
-  (model/*:model*/, action/*:action*/)/*:[model, Effects<action>]*/ => {
-    console.warn('Unknown action was passed & ignored: ', action, Error().stack);
-    return [model, Effects.none];
-  };
```

Comment:
Just curios what was the issue with:

```js
export const update = <model, action>
  (model:model, action:action):[model, Effects<action>] => {
    console.warn('Unknown action was passed & ignored: ', action, Error().stack);
    return [model, Effects.none];
  };
```

Asking because it supposed to and if it does not maybe a babel or our configuration issue & if former I'd like to submit a bug.